src/main/java/io/voucherify/client/model/voucher/DiscountEffect.java

```diff
@@ -6,5 +6,6 @@ public enum DiscountEffect {
   APPLY_TO_ORDER,
   APPLY_TO_ITEMS,
   APPLY_TO_ITEMS_PROPORTIONALLY,
-  APPLY_TO_ITEMS_PROPORTIONALLY_BY_QUANTITY
+  APPLY_TO_ITEMS_PROPORTIONALLY_BY_QUANTITY,
+  APPLY_TO_EVERY
```

Comment:
I'm using it with if the validate method client.validations().validate(voucherCode, validation);

Reply:
And are You sure this method returns `APPLY_TO_EVERY` under `discount.effect` key? Because i see that it returns `APPLY_TO_EVERY` under `applicable_to.data[index].effect` and above change are not related to this key but only for class Discount and class DiscountEffect which are strict connected to `discount.effect` key

Reply:
Yes, my code looks like that

 ```
       final var response = client.validations().validate(voucherCode, validation);
        if (response.isValid()) {
            final DiscountEffect discountEffect;
            if (response.getGift() != null) {
                discountEffect = DiscountEffect.valueOf(response.getGift().getEffect().name());
            } else
                discountEffect = response.getDiscount().getEffect();

             if (discountEffect == DiscountEffect.APPLY_TO_ORDER) {
             } else if ((discountEffect == DiscountEffect.APPLY_TO_ITEMS) || (discountEffect == DiscountEffect.APPLY_TO_EVERY)) {
                    }
        }

```

Reply:
And can You show the request response? Through API (postman etc.) or SDK parsed to string. Because I'm not sure that DiscountEffect.APPLY_TO_EVERY is possible in response.

Reply:
I consulted it with my team and we double checked that `APPLY_TO_EVERY` is not occur under `discount.effect` key (in DiscountEffect class). You could confuse it with e.g `redeemables.applicable_to.data[index].effect` which is correlated to validation rules and there could be `APPLY_TO_EVERY` const. This could be confusing and probably will be detailed in future versions. For now it works in followed way.
`discount.effect` answers the question `What is the target of the discount? Order or items?`
`applicable_to` answers the question `Does the discount apply to every item or maybe the cheapest/most expensive?`